doc/cascadia/profiles.schema.json

```diff
@@ -396,7 +396,7 @@
         },
         "rowsToScroll": {
           "default": "system",
-          "description": "The number of rows to scroll at a time with the mouse wheel. This will override the system setting if the value is not zero or \"system\".",
+          "description": "This parameter has been deprecated.",
```

Comment:
Is there a built-in way to deprecate things through the JSON schema? I tried looking into it but I got a bit lost.

Reply:
Not that I’m aware of.

Reply:
Maybe, but I don’t know if we’re using a new-enough version of the json-schema format